db/migrate/20200803075009_migrate_host_config_groups_to_facet.foreman_puppet_enc.rb

```diff
@@ -0,0 +1,8 @@
+class MigrateHostConfigGroupsToFacet < ActiveRecord::Migration[6.0]
+  def change
+    HostConfigGroup.unscoped.in_batches do |batch|
+      host_ids = batch.uniq.pluck(:host_id)
+      HostPuppetFacet.create(host_ids.map{|host_id| {host_id: host_id}})
```

Comment:
this will create appropriate facets, but now we will need to assign the `HostConfigGroup` to this facet.
Also this migration will mean we can't merge this first.
Could we move this migration to the models PR?